client/src/entities/revision-meta.js

```diff
@@ -0,0 +1,17 @@
+import { Record } from 'immutable';
+import Diff from './diff';
+
+export default class RevisionMeta extends Record( {
+	status: 'done',
+	error: undefined,
+	diff: new Diff()
```

Comment:
I don't think diffs belong inside revision. Specially since diffs deal with two revisions. Moving them out will also remove the long nested objects like `action.revision.meta.diff.meta.[...]` deal.

Reply:
I went back and forth on this. I can see pros and cons both ways. I think I would prefer for them to be separate as well, but it felt more complex, but yeah since it's a different object, it should probably be in a different list.